analysis/tc_acmeair_webapp_upload_binary.go

```diff
@@ -12,14 +13,417 @@ var AcmeairWebappBinary = TC{
 	Task:        Analyze,
 	Labels: addon.Labels{
 		Included: []string{
-			"konveyor.io/target=cloud-readiness",
+			"konveyor.io/target=jakarta-ee",
 		},
 	},
 	Binary:   true,
 	Artifact: "/binary/acmeair-webapp-1.0-SNAPSHOT.war",
+	Identities: []api.Identity{
+		identity.TackleTestappPublicMaven,
```

Comment:
There are just refs to public maven artifacts, but it feel better to me to have maven central repository explicitly stated in the settings.xml file (provided by the TackleTestappPublicMaven identity)

Reply:
Upload binary analysis doesn't need credentials, could you please remove this?

Reply:
Makes sense, will remove.

Reply:
settings.xml contains much more than credentials, in this case repository definition, I found this part https://github.com/konveyor/go-konveyor-tests/blob/main/data/identity/maven.go#L84-L108 to provide some help when debugging mvn command failures, but don't have string opinion, we can go with defaults without maven settings identity resource too.